engine/schema/src/main/resources/META-INF/db/schema-41310to41400-cleanup.sql

```diff
@@ -25,4 +25,4 @@ DELETE FROM `cloud`.`configuration` WHERE name = 'host.maintenance.retries';
 UPDATE `cloud`.`storage_pool` SET path="/var/lib/libvirt/images" WHERE path="/var/lib/libvirt/images/";
 
 -- remove (one of) duplicate unique indexes from Region table
-ALTER TABLE `region` DROP INDEX `id_3`;
+ALTER TABLE `region` DROP INDEX IF EXISTS `id_3`;
```

Comment:
MySQL does not support `DROP INDEX IF EXISTS`, at least as far as I know there is no version supporting. If there is, can you please point it here?

Reply:
I ran this, it worked:
```
> ALTER TABLE `region` DROP INDEX IF EXISTS `id_3`;
Query OK, 0 rows affected, 1 warning (0.002 sec)
Records: 0  Duplicates: 0  Warnings: 1

```

Reply:
@rhtyd are yiou using mongoDB or mariaDB?

I just execcuted it on a Mysql and it indeed fails:
```
myysql> ALTER TABLE `region` DROP INDEX IF EXISTS `id_3`;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'IF EXISTS `id_3`' at line 1
```

Reply:
Mysql documentation (https://dev.mysql.com/doc/refman/5.7/en/drop-index.html):
```
DROP INDEX [ONLINE | OFFLINE] index_name ON tbl_name
    [algorithm_option | lock_option] ...

algorithm_option:
    ALGORITHM [=] {DEFAULT | INPLACE | COPY}

lock_option:
    LOCK [=] {DEFAULT | NONE | SHARED | EXCLUSIVE}
```

MariaDB:
```
DROP INDEX [IF EXISTS] index_name ON tbl_name 
    [WAIT n |NOWAIT]
    [algorithm_option | lock_option] ...

algorithm_option:
    ALGORITHM [=] {DEFAULT|INPLACE|COPY|NOCOPY|INSTANT}

lock_option:
    LOCK [=] {DEFAULT|NONE|SHARED|EXCLUSIVE}
```